packages/trigger-pager/triggerPager.js

```diff
@@ -0,0 +1,54 @@
+const { api } = require('@pagerduty/pdjs')
+
+const PRIORITY_P1 = 'PUTY3A1'
+const DUPLICATE_INCIDENT_MESSAGE = 'matching dedup key already exists'
+
+const triggerPager = async ({
+  token,
+  serviceId,
+  incidentKey,
+  urgency = 'high',
+  from = 'billing@transloadit.com',
+  title = incidentKey,
+  description,
+}) => {
+  const res = await api({ token }).post('/incidents', {
+    headers: {
+      from,
+    },
+    data: {
+      incident: {
+        type        : 'incident',
+        incident_key: incidentKey,
+        urgency,
+        title       : title.length >= 1024
```

Comment:
You could use abbr, also in this repo. It abbreviated in the middle, which often can often be helpful-ish as identifiers or counters often come last

Reply:
Good tip! Let's add that when it becomes relevant, I just wanna get this out 😎